docs/source/reference/comparison.rst

```diff
@@ -0,0 +1,172 @@
+
+   run: |
+     conda deactivate
+     echo 'Starting vllm openai api server...'
+     python -m vllm.entrypoints.openai.api_server \
+     --model $MODEL_NAME \
+     --tokenizer hf-internal-testing/llama-tokenizer \
+     --host 0.0.0.0
+           </code>
+           </pre>
+       </div>
+       <div class="col-md-6 mb-3">
+            <h3> Kubernetes </h3>
+           <pre>
+           <code class="yaml">
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+   name: vllm-gemma-deployment
+   spec:
+   replicas: 1
+   selector:
+     matchLabels:
+       app: gemma-server
+   template:
+     metadata:
+       labels:
+         app: gemma-server
+         ai.gke.io/model: gemma-1.1-2b-it
+         ai.gke.io/inference-server: vllm
+         examples.ai.gke.io/source: user-guide
+     spec:
+       containers:
+       - name: inference-server
+         image: us-docker.pkg.dev/vertex-ai/ vertex-vision-model-garden-dockers/pytorch-vllm-serve:20240527_0916_RC00
+         resources:
+           requests:
+             cpu: "2"
+             memory: "10Gi"
+             ephemeral-storage: "10Gi"
+             nvidia.com/gpu: 1
+           limits:
+             cpu: "2"
+             memory: "10Gi"
+             ephemeral-storage: "10Gi"
+             nvidia.com/gpu: 1
+         args:
+         - --model=$(MODEL_ID)
+         - --tensor-parallel-size=1
+         env:
+         - name: MODEL_ID
+           value: google/gemma-1.1-2b-it
+         - name: HUGGING_FACE_HUB_TOKEN
+           valueFrom:
+             secretKeyRef:
+               name: hf-secret
+               key: hf_api_token
+         volumeMounts:
+         - mountPath: /dev/shm
+           name: dshm
+       volumes:
+       - name: dshm
+         emptyDir:
+             medium: Memory
+       nodeSelector:
+         cloud.google.com/gke-accelerator: nvidia-l4
+   ---
+   apiVersion: v1
+   kind: Service
+   metadata:
+   name: llm-service
+   spec:
+   selector:
+     app: gemma-server
+   type: ClusterIP
+   ports:
+     - protocol: TCP
+       port: 8000
+       targetPort: 8000
+           </code>
+           </pre>
+       </div>
+   </div>
+
+
+Scale beyond single region/cluster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: https://blog.skypilot.co/ai-on-kubernetes/images/failover.png
+    :align: center
+    :width: 80%
+    :alt: Scaling beyond a single region Kubernetes cluster with SkyPilot
+
+    If the Kubernetes cluster is full, SkyPilot can get GPUs from other regions and clouds to run your tasks at the lowest cost.
+
+Kubernetes deployments are typically contained to a single region in a single cluster.
+This is because longer latencies across regions causes etcd, the store for Kubernetes state, to timeout and fail [1]_ [2]_ [3]_.
+
+Being restricted to a single region/cloud with Vanilla Kubernetes has two drawbacks:
+
+1. `Availability is reduced by up to 29% <https://blog.skypilot.co/introducing-sky-serve/#why-skyserve>`_ because you cannot utilize
```

Comment:
```suggestion
1. `GPU availability is reduced by up to 29% <https://blog.skypilot.co/introducing-sky-serve/#why-skyserve>`_ because you cannot utilize
```
Actually, this statement & 29% is a bit odd at face value. One would think by going to many more regions + clouds the GPU availability can intuitive more than 3x. Similar thoughts for 2.4x below. Shall we skip the numbers?

Reply:
Removed the numbers